packages/js/product-editor/package.json

```diff
@@ -73,6 +73,7 @@
 		"@wordpress/block-editor": "^9.8.0",
 		"@wordpress/browserslist-config": "wp-6.0",
 		"concurrently": "^7.0.0",
+		"copy-webpack-plugin": "9",
```

Comment:
Maybe just update this to follow a similar format as the rest, like `^9.0.0`?
It also seems like there is a discrepancy between the package.json and the `pnpm-lock` file, so running a `pnpm i` and pushing that up should help.

Reply:
I ran `pnpm --filter=@woocommerce/product-editor add -D copy-webpack-plugin@^9.0.0` which set it to `^9.1.0` due to the `^`.

I'm a little afraid of manually changing this in the package file since this has caused us some frustration with pnpm in the past.